image-metadata/metadata/Acronyms.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Acronyms && Descriptions
+date: today
+author: Contributors
+---
+
+# ***Acronyms***
+
+![Acronyms](https://img.shields.io/badge/BENCHMARK-Acronyms-FFFC33)
+
+<br/><br/> 
+
+
+## **Techniques**
+
+| Abbreviation      | Description | 
+| ---------------- | ------------ | 
+| AET | Analytical Electron Microscopy |
+| AFM | Atomic Force Microscopy | 
+| AT_TEM | Array Tomography Transmission Electron Microscope |
+| ATUM_SEM | Automated Tape Collecting Ultramicrotome Scanning Electron Microscope | 
+| CLARITY | **C**lear **L**ipid-exchanged **A**crylamide-hybridized **R**igid **I**maging / **I**mmunostaining / in situ-hybridization-compatible **T**issue h**Y**drogel |
+| Cryo_FIB_SEM | Cryogenic focused Ion Beam-Scanning Electron Microscope |
+| FIB_SEM | Focused Ion Beam-scanning Electron microscope |
+| LEEM | Low-Energy Electron Microscopy |
+| LM | Light Microscopy |
+| PEEM | Photoemission Electron Microscopy |
+| mAT | Magneto Acoustic Tomography |
+| MRI | Magnetic Resonance Imaging |
+| REM | Reflection Electron Microscope |
+| sAT | Scanning Acoustic Tomography |
+| SB_SEM | Serial Block-Face Scanning Electron Microscope |
+| SEM | Scanning Electron Microscope |
+| S_TEM | Scanning Transmission electron Microscopy |
+| SS_TEM | Serial Section Transmission Electron Microscopy |
+| STM | Scanning Tunneling Microscope |
+| TEM | Transmission Electron Microscope |
+| Xray | X-Radiation |
+| XRM | Xray Microscopy |
```

Comment:
@satra do you know any existing ontology/vocab which defines all those and with acronyms already?  I found some part of https://www.ebi.ac.uk/ols/ontologies/fbbi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FFBbi_00000258 but seems lacking some (e.g. MRI)

Reply:
the techniques and modalities ontology (https://github.com/BICCN/TMN/tree/90cf0bdd1756af1cf26d0f8d837b4a9df2a4b96d) covers  several but is not exhaustive.

Reply:
and this covers others: https://www.ebi.ac.uk/ols4/ontologies/edam/classes/http%253A%252F%252Fedamontology.org%252Ftopic_3361?lang=en